### PR TITLE
Startup/AppChooser: misc cleanup

### PR DESCRIPTION
**Pull request by danrabbit**
* Bump copyright
* Signals at the top
* Explicit scope
* Code style
* Remove single-use function

### CODE REVIEW

File: src/Startup/Widgets/AppChooser.vala [hunk 1]
```diff
@@ -20,96 +20,94 @@
...
+        };
+        grid.attach (search_entry, 0, 0);
+        grid.attach (scrolled, 0, 1);
+        grid.attach (custom_entry, 0, 2);
 
         add (grid);
 
         search_entry.grab_focus ();
+        search_entry.search_changed.connect (() => {
+            list.set_filter_func (filter_function);
```
Comment (tintou) [inline src/Startup/Widgets/AppChooser.vala:74]:
```suggestion
            list.invalidate_filter ();
```

File: src/Startup/Widgets/AppChooser.vala [hunk 2]
```diff
@@ -20,96 +20,94 @@
...
-        list.prepend (app_row);
-    }
-
-    int sort_function (Gtk.ListBoxRow list_box_row_1,
-                       Gtk.ListBoxRow list_box_row_2) {
-        var row_1 = list_box_row_1.get_child () as AppChooserRow;
-        var row_2 = list_box_row_2.get_child () as AppChooserRow;
+    private int sort_function (Gtk.ListBoxRow row1, Gtk.ListBoxRow row2) {
+        var row_1 = row1.get_child () as AppChooserRow;
+        var row_2 = row2.get_child () as AppChooserRow;
```
Comment (tintou) [inline src/Startup/Widgets/AppChooser.vala:90]:
```suggestion
        unowned AppChooserRow row_1 = row1.get_child () as AppChooserRow;
        unowned AppChooserRow row_1 = row1.get_child () as AppChooserRow;
```
```suggestion
        unowned AppChooserRow row_1 = (AppChooserRow) row1.get_child ();
        unowned AppChooserRow row_2 = (AppChooserRow) row2.get_child ();
```

File: src/Startup/Widgets/AppChooser.vala (hunk 1 above)
Comment (danrabbit) [reply src/Startup/Widgets/AppChooser.vala:74]: duh, dunno how I didn't catch this

**Review by jeremypw — APPROVED** [review]
Works as expected, no regressions found

**Pull request merged**